app/src/main/java/org/simple/clinic/introvideoscreen/IntroVideoScreen.kt

```diff
@@ -106,7 +106,7 @@ class IntroVideoScreen(
   }
 
   override fun openHome() {
-    val screenKey = screenKeyProvider.keyFor<IntroVideoScreenKey>(this)
+    val screenKey = screenKeyProvider.keyFor<Key>(this)
```

Comment:
Just an FYI, we don't need to replace it here. When we extend the `BaseScreen` we can get the `screenKey` variable from the screen instead of the provider. So we would end up removing this piece of code.